packages/connect-webextension/src/popup.ts

```diff
@@ -0,0 +1,162 @@
+// origin: https://github.com/trezor/connect/blob/develop/src/js/popup/PopupManager.js
+import EventEmitter from 'events';
+import { PopupEventMessage } from 'packages/connect/lib';
```

Comment:
This should probably be `@trezor/connect/lib`

Reply:
oh for sure!